src/util/charmrun-src/charmrun.C

```diff
@@ -1185,7 +1185,7 @@ static void arg_init(int argc, const char **argv)
 #if CMK_SMP
     if (arg_requested_pes > 0 && arg_requested_nodes > 0 && arg_ppn > 0 && arg_ppn * arg_requested_nodes != arg_requested_pes)
     {
-      fprintf(stderr, "Charmrun> Error: ++np times ++ppn does not equal +p.\n");
+      fprintf(stderr, "Charmrun> Error: +n/++np %d times ++ppn %d does not equal +p %d.\n", arg_requested_nodes, arg_ppn, arg_requested_pes);
```

Comment:
Should be ++n instead of +n.

Reply:
+n with a single plus is supported, like +p. I don't want to change all mentions of +p and +n to also say ++p and ++n.

Reply:
OK, thanks.